tests/integration-tests/tests/networking/test_cluster_networking/test_cluster_in_no_internet_subnet/osu_pt2pt_submit_openmpi.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#SBATCH --ntasks=2 --nodes=2
+
+set -e
+
+module load openmpi
+mpirun --map-by ppr:1:node /shared/openmpi/osu-micro-benchmarks-{{ osu_benchmark_version }}/mpi/pt2pt/osu_latency
```

Comment:
probably you can get rid of `--map-by ppr:1:node` cause mpi should get this from Slurm. Not sure whether you can do the same with intelmpi